src/version/reducers/feedsListReducer.js

```diff
@@ -0,0 +1,41 @@
+import { get } from "lodash";
+
+import { FEEDS_LIST_ACTION } from "../actions/feedsListAction";
+
+const initialState = {
+    data: [],
+    loading: false,
+    error: null,
+};
+
+/**
+ * This component returns reducer for Feeds list action
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @param {shape} state
+ * @param {shape} action
+ * @return {shape}
+ */
+export default (state = initialState, action) => {
+    switch (action.type) {
+        case FEEDS_LIST_ACTION.REQUEST:
+            return {
+                ...state,
+                loading: true,
+            };
+        case FEEDS_LIST_ACTION.SUCCESS:
+            return {
+                ...state,
+                loading: false,
+                data: get(action, "data", []),
+            };
+        case FEEDS_LIST_ACTION.FAILURE:
+            return {
+                ...state,
+                loading: false,
+                error: get(action, "error", null),
```

Comment:
you catch the error but you never read it in the components...

Reply:
PulseTile doesn't have alerts about success or error during operations. But this possibility can be added in future. And I want to have possibility to extract error message in future. 

It isn't difficult to remove, but can I leave it?

Reply:
Sure, but the longer you wait for it, the herder it gets. In my opinion, when the error cases aren't handled, a feature is not finished.